src/test/resources/features/Registration.feature

```diff
@@ -4,7 +4,8 @@ Feature: Register for ECL
   Scenario Outline: User registers a <Entity type> supervised by HMRC for AML that is liable for ECL
     Given I am signed in to the registration journey
     When I provide details of my limited company that is supervised by HMRC and liable for ECL
-    Then I should be on the placeholder page that says Success - you can continue registering for ECL
+    And I select No for whether or not I started AML regulated activity in current FY
```

Comment:
I thought we were putting these in the step def as part of the `provide` method, to maintain declarative style?